arch/xtensa/src/common/xtensa_createstack.c

```diff
@@ -73,6 +73,18 @@
 #define STACK_ALIGN_DOWN(a) ((a) & ~STACK_ALIGN_MASK)
 #define STACK_ALIGN_UP(a)   (((a) + STACK_ALIGN_MASK) & ~STACK_ALIGN_MASK)
 
+#ifdef CONFIG_XTENSA_USE_SEPERATE_IMEM
```

Comment:
I see these macros are used in some places, and they are defined in multi `*.c` files, so maybe we can move these into a new file like `xtensa_mm.h` ,and let these `*.c` include `xtensa_mm.h`.

Reply:
These macros come from config.h (which is generated based on the .config file created by Kconfig tools)
*.c files include the config.h and use the macros.

Reply:
Ah I see what macros you are talking about, Github confused me.

Reply:
@donghengqaz done.  PTAL.